ext-meta-generator/src/test/resources/output/KotlinDataClassFields.java

```diff
@@ -0,0 +1,20 @@
+package input.generated;
+
+import javax.annotation.processing.Generated;
+
+@Generated("tech.ydb.yoj.generator.FieldGeneratorAnnotationProcessor")
+public final class KotlinDataClassFields {
+    private KotlinDataClassFields(){}
+
+    public static final String FIELD1 = "field1";
+    public static final String FIELD2 = "field2";
+    public static final String STATUS = "status";
+    public static final String DATA = "data";
+    public static final String ID_INTEGRAL = "id";
```

Comment:
The same reason why other complex fields have the preffix - to avoid name clashing and make it clear for a user (that they reference to the Id as a whole object rather than just a field)

The id was not got collapsed into a single ID field because it has 2 fields inside of it